packages/fxa-content-server/app/scripts/templates/support.mustache

```diff
@@ -100,4 +102,4 @@
   </div>
 </div>
 
-<footer id="legal-footer"><a class="terms" href="/legal/terms">{{#t}}Terms of Service{{/t}}</a><a class="privacy" href="/legal/privacy">{{#t}}Privacy Notice{{/t}}</a></footer>
+<footer id="legal-footer"><a class="terms text-[#0060df] no-underline" href="/legal/terms">{{#t}}Terms of Service{{/t}}</a><a class="privacy text-[#0060df] no-underline" href="/legal/privacy">{{#t}}Privacy Notice{{/t}}</a></footer>
```

Comment:
Not sure if we need to change the styling in ./security_events.mustache to match as well. It was the only other result I saw for `id="legal-footer"`

```sh
git grep -n ' id="legal-footer"' packages | cat

packages/fxa-content-server/app/scripts/templates/security_events.mustache:26: <footer id="legal-footer"><a class="terms" href="/settings?security_events=true">{{#t}}Back to settings{{/t}}</a></footer>
packages/fxa-content-server/app/scripts/templates/support.mustache:105:        <footer id="legal-footer"><a class="terms text-[#0060df] no-underline" href="/legal/terms">{{#t}}Terms of Service{{/t}}</a><a class="privacy text-[#0060df] no-underline" href="/legal/privacy">{{#t}}Privacy Notice{{/t}}</a></footer>
```

Although if I search more generically for "Terms of Service" it looks like we're generally using `link-grey` (or `link-blue` in some mjml emails).

Reply:
I've added the `link-blue` class to `security_events.mustache`. Thank you for finding this.